geode-core/src/main/java/org/apache/geode/internal/statistics/platform/LinuxProcFsStatistics.java

```diff
@@ -156,14 +152,24 @@ public static void refreshProcess(int pid, LocalStatisticsImpl stats) {
   }
 
   public static void refreshSystem(LocalStatisticsImpl stats) {
+    try {
+      refreshSystem(stats, new FileInputStream("/proc/stat"),
+          new FileInputStream("/proc/net/netstat"));
```

Comment:
I think that the potential resource leaks here can be fixed by using try with resources:
```
try (FileInputStream procStat = new FileInputStream("/proc/stat");
FileInputStream netStat = new FileInputStream("/proc/net/stat")) {
    refreshSystem(stats, procStat, netStat);
}
```

Reply:
I ended up changing refreshSystem to take in file paths to have only one try/catch.  There was a scenario I thought of where if I kept the try and it failed to FileInputStream, it wouldn't update the other stats.  The original code would attempt to update the other stats if the FileInputStream failed

Reply:
Ah, yeah, that's a nicer approach